documentation/install.rst

```diff
@@ -222,19 +222,25 @@ If ``binder`` was build with some older versions of LLVM, one could also set the
 
 .. _building-static:
 
+With Docker
+***********
+
+A `Dockerfile` for building binder can be found in the ``binder`` repository linked here: https://github.com/RosettaCommons/binder/
```

Comment:
Probably best to use relative path here since someone could be using local-clone or fork of this repository. Could you please update this? Thanks,

Reply:
I'm not exactly familiar with whatever `.rst` is. I presume you mean replace `https://github.com/RosettaCommons/binder/` with `SOMETHING` ? If you tell me what the relative path would be I can replace it in the PR though, though I'm not sure what it relative to exactly since this will be hosted on readthedocs, no? Should the replacement string be `..` or something else?

Reply:
oh, i was not suggesting to make this into real link (it will be challenging since documentation could be on different web site). Ok, how about just replacing this link with `https://github.com/RosettaCommons/binder/examples` (assuming that we are moving Dockerfile into examples dir). Thanks,